src/global/tracking/tracking.cc

```diff
@@ -28,149 +29,129 @@
 
 //
 extern "C" {
-void InitPlugin(JApplication *app) {
```

Comment:
Change the settings in your IDE to avoid changing spacing in files when you don't change the content. Change the spacing so this only shows the differences that are actually relevant.